cspell.yaml

```diff
@@ -19,6 +19,7 @@ dictionaries:
 useGitignore: true
 ignorePaths:
   - '**/generated/**'
+  - 'packages/usage-reporting-protobuf/src/reports.proto'
```

Comment:
I guess flipside here is, if there are typos in this file then we should fix them at the source? (Perhaps with a temporary addition to local dictionary?)

Reply:
Maybe we do want to permit `nestedness` and catch future spelling problems here 🤷